configuration/pih/pih-config-haiti.json

```diff
@@ -15,6 +15,17 @@
         "registrationapp.paperRecordIdentifierDefinition": "paperrecord.patientDataDefinition.paperRecordIdentifier"
     },
 
+    "authenticationConfig": {
+      "scheme": "2fa",
+      "schemes": {
+        "2fa": {
+          "config": {
+            "secondaryOptions": "secret"
```

Comment:
totp should also be added, unless there is a specific reason we are choosing not to include it @mogoodrich ?  why do you think it doesn't work?

Reply:
Perfect, I'll add it back it... no reason to think it doesn't work, I just had forgotten if we had built it or not.